### DISCUSSION TITLE
full width tabulator?

### DISCUSSION THREAD

**Discussion by kushfest** [anvilistas/tabulator#48] [Q&A]
For the life of me I can't seem to get any of my tabulator grids to show as full width. I've read everything I can find on the topic in the anvil forums, and it seems its should be as simple as making sure that both the tabulator component and the column panel it is in, should have full_row_width set to True. I've carefully checked, and that is the case, but I still am not getting a tabulator grid that fills the space on the screen.

Any suggestions as to what else I can try? Happy to share a clone link directly, but would prefer not to post it.

**Comment by s-cork** [comment]
Sure just drop me a message on the forum.
